tests/tracepoint_format_parser.cpp

```diff
@@ -1,150 +1,153 @@
-#include "gtest/gtest.h"
 #include "tracepoint_format_parser.h"
+#include "gtest/gtest.h"
 
 namespace bpftrace {
 namespace test {
 namespace tracepoint_format_parser {
 
-class MockTracepointFormatParser : public TracepointFormatParser
-{
+class MockTracepointFormatParser : public TracepointFormatParser {
 public:
-  static std::string get_tracepoint_struct_public(std::istream &format_file, const std::string &category, const std::string &event_name)
-  {
+  static std::string
+  get_tracepoint_struct_public(std::istream &format_file,
+                               const std::string &category,
+                               const std::string &event_name) {
     return get_tracepoint_struct(format_file, category, event_name);
   }
 };
 
-TEST(tracepoint_format_parser, tracepoint_struct)
-{
+TEST(tracepoint_format_parser, tracepoint_struct) {
   std::string input =
-    "name: sys_enter_read\n"
-    "ID: 650\n"
-    "format:\n"
-    "	field:unsigned short common_type;	offset:0;	size:2;	signed:0;\n"
-    "	field:unsigned char common_flags;	offset:2;	size:1;	signed:0;\n"
-    "	field:unsigned char common_preempt_count;	offset:3;	size:1;	signed:0;\n"
-    "	field:int common_pid;	offset:4;	size:4;	signed:1;\n"
-    "\n"
-    "	field:int __syscall_nr;	offset:8;	size:4;	signed:1;\n"
-    "	field:unsigned int fd;	offset:16;	size:8;	signed:0;\n"
-    "	field:char * buf;	offset:24;	size:8;	signed:0;\n"
-    "	field:size_t count;	offset:32;	size:8;	signed:0;\n"
-    "\n"
-    "print fmt: \"fd: 0x%08lx, buf: 0x%08lx, count: 0x%08lx\", ((unsigned long)(REC->fd)), ((unsigned long)(REC->buf)), ((unsigned long)(REC->count))\n";
-
-  std::string expected =
-    "struct _tracepoint_syscalls_sys_enter_read\n"
-    "{\n"
-    "  unsigned short common_type;\n"
-    "  unsigned char common_flags;\n"
-    "  unsigned char common_preempt_count;\n"
-    "  int common_pid;\n"
-    "  int __syscall_nr;\n"
-    "  u64 fd;\n"
-    "  char * buf;\n"
-    "  size_t count;\n"
-    "};\n";
+      "name: sys_enter_read\n"
+      "ID: 650\n"
+      "format:\n"
+      "	field:unsigned short common_type;	offset:0;	size:2;	"
+      "signed:0;\n"
```

Comment:
Why's it chosen to split this string? The line isn't over 80 characters even with the extra indentation

Reply:
Not sure. But we can turn off clang-format as necessary with
```
// clang-format off
<blah>
// clang-format on
```